apps/tng-dictionary/src/components/Carousel/Carousel.tsx

```diff
@@ -50,51 +50,39 @@ export function Carousel(props: CarouselProps) {
 
   return (
 
-    <div>
-      <AnimatePresence>
 
-        <Card className="Cards">
-
-          <CardContent>
-            <motion.div key={currentIndex} initial={{ x: isToggled }} animate={{ x: 0, y: 0, scale: 1, rotate: 0, }} exit={{ x: 12, y: 0, scale: 1, rotate: 0, }} transition={{ duration: 1 }}>
-              <Term termData={currentItem} />
-              <Divider sx={{ background: 'rgb(150, 150, 150)' }} />
+    <AnimatePresence>
+
+      <Card className="Cards">
+        <CardContent>
+          <motion.div key={currentIndex} initial={{ x: isToggled }} animate={{ x: 0, y: 0, scale: 1, rotate: 0, }} exit={{ x: 12, y: 0, scale: 1, rotate: 0, }} transition={{ duration: 1 }}>
+            <Term termData={currentItem} />
+            <Divider className='Divider' />
+          </motion.div>
+
+          <div className='carouselButtons'>
+            <motion.div onClick={() => setToggle(prevValue => {
+              return prevValue ? -200 : -200;
+            })} whileHover={{ scale: 1.2, transition: { duration: 0.1 } }}
+              whileTap={{ scale: 0.9 }}>
+              <Button className='clicker' disableRipple={true} onClick={e => setIndex(cyclicDecrement(currentIndex, props.data.length - 1))}><ArrowBackIosNewOutlinedIcon /> Back</Button>
```

Comment:
The lines of code are pretty long. Is prettier doing its job? If this is how the formatter renders it, we should probably breakout the `onClick` into a separate function
`onClick={e => handleClickPrevious()}`
not sure if we could go further and make that point free (i.e.
`onClick={handleClickPrevious}`
given that `e` is never actually used. Give it a try.